scraper/__main__.py

```diff
@@ -259,6 +262,7 @@ def scrape(
                         os.path.join(case_html_path, f"{case_id}.html"), "w"
                     ) as file_handle:
                         file_handle.write(case_html)
+                    write_to_blob(os.path.join(case_html_path, f"{case_id}.html"), case_id)

```

Comment:
We can probably use a BytesIO stream here and pass that directly to the blob, rather than writing to the file in an intermediate step

Reply:
That sounds good, I second this